src/custom/pages/Faq/index.tsx

```diff
@@ -373,13 +384,19 @@ export default function Faq() {
 
           <p>
             Yes, you can directly place buy and sell orders for ETH. Before the actual order is placed, the UI will
-            allow you to wrap and unwrap ETH into WETH without needing to leave the dapp’s UI.{' '}
+            allow you to wrap and unwrap ETH into WETH without needing to leave the dapp’s UI
           </p>
 
           <hr />
 
           <p>
-            Didn&#39;t find an answer? Join the <a href={DISCORD_LINK}>community on Discord</a> for support.
+            Didn&#39;t find an answer? Join the <a href={DISCORD_LINK}>community on Discord</a>
+          </p>
+          <p>
+            We really hope you like CowSwap. If you do,&nbsp;<Link to="/">Milk it!</Link>
+            <span role="img" aria-label="happy cow face">
```

Comment:
@anxolin 
Update aria-label to match the emoji ;)

Reply:
🙈 ahahah